### PR TITLE
EJ2-72229 : How to change the signature indicator text

### PR DESCRIPTION
**Pull request by SaravanaPriya31**
### Description 
How to change the signature indicator text

### CODE REVIEW

**Review by florenceliliana — CHANGES_REQUESTED** [review]

File: ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/change-signature-indicator-text.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+---
+layout: post
```
Comment (Chrispineagunja) [inline ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/change-signature-indicator-text.md:2]: Payout: Post

File: ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/change-signature-indicator-text.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+---
+layout: post
+title: How to change the signature indicator settings | Syncfusion
```
Comment (Chrispineagunja) [inline ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/change-signature-indicator-text.md:3]: Title: How to change the signature indicator settings | Syncfusion

File: ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/change-signature-indicator-text.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
+---
+layout: post
+title: How to change the signature indicator settings | Syncfusion
+description: Learn how to change the signature indicator settings in Syncfusion ##Platform_Name## Pdfviewer component of Syncfusion Essential JS 2 and more.
```
Comment (Chrispineagunja) [inline ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/change-signature-indicator-text.md:4]: Description: Learn how to change the signature indicator settings in Syncfusion ##Platform_Name## Pdfviewer component of Syncfusion Essential JS 2 and more.

File: ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/change-signature-indicator-text.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
+---
+layout: post
+title: How to change the signature indicator settings | Syncfusion
+description: Learn how to change the signature indicator settings in Syncfusion ##Platform_Name## Pdfviewer component of Syncfusion Essential JS 2 and more.
+platform: ej2-asp-core-mvc
```
Comment (Chrispineagunja) [inline ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/change-signature-indicator-text.md:5]: Platform: ej2-asp-core-mvc

File: ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/change-signature-indicator-text.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
+---
+layout: post
+title: How to change the signature indicator settings | Syncfusion
+description: Learn how to change the signature indicator settings in Syncfusion ##Platform_Name## Pdfviewer component of Syncfusion Essential JS 2 and more.
+platform: ej2-asp-core-mvc
+control: How to change the signature indicator settings
```
Comment (Chrispineagunja) [inline ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/change-signature-indicator-text.md:6]: Control: How to change the signature indicator settings

File: ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/change-signature-indicator-text.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
+---
+layout: post
+title: How to change the signature indicator settings | Syncfusion
+description: Learn how to change the signature indicator settings in Syncfusion ##Platform_Name## Pdfviewer component of Syncfusion Essential JS 2 and more.
+platform: ej2-asp-core-mvc
+control: How to change the signature indicator settings
+publishingplatform: ##Platform_Name##
```
Comment (Chrispineagunja) [inline ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/change-signature-indicator-text.md:7]: Publishingplatform: ##Platform_Name##

File: ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/change-signature-indicator-text.md [hunk 7]
```diff
@@ -0,0 +1,48 @@
+---
+layout: post
+title: How to change the signature indicator settings | Syncfusion
+description: Learn how to change the signature indicator settings in Syncfusion ##Platform_Name## Pdfviewer component of Syncfusion Essential JS 2 and more.
+platform: ej2-asp-core-mvc
+control: How to change the signature indicator settings
+publishingplatform: ##Platform_Name##
+documentation: ug
```
Comment (Chrispineagunja) [inline ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/change-signature-indicator-text.md:8]: Documentation: UG

File: ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/change-signature-indicator-text.md [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+description: Learn how to change the signature indicator settings in Syncfusion ##Platform_Name## Pdfviewer component of Syncfusion Essential JS 2 and more.
+platform: ej2-asp-core-mvc
+control: How to change the signature indicator settings
+publishingplatform: ##Platform_Name##
+documentation: ug
+---
+
+# How to change the signature indicator settings
+
+To change the signature indicator settings in the Syncfusion PDF Viewer, you need to use the `signatureIndicatorSettings` property of the `signatureFieldSettings` object.You can modify various properties of the signature indicator such as opacity, background color, width, height, font size, text, and color.
```
Comment (Chrispineagunja) [inline ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/change-signature-indicator-text.md:13]: To change the signature indicator settings in the Syncfusion PDF Viewer, use the `signatureIndicatorSettings` property of the `signatureFieldSettings` object. Modify various properties of the signature indicator such as opacity, background color, width, height, font size, text, and color.

File: ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/change-signature-indicator-text.md [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+
+Here's an example of how to change the signature indicator settings
```
Comment (Chrispineagunja) [inline ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/change-signature-indicator-text.md:15]: The following is an example of how to change the signature indicator settings.

File: ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/change-signature-indicator-text.md [hunk 10]
```diff
@@ -0,0 +1,48 @@
...
+            fontSize: 10, 
+            text: 'Your Text Here', 
+            color: 'blue' 
+        };
+    }
+</script>
+
+```
+
+If you want to change any of these properties, you can simply modify the corresponding value in the `signatureIndicatorSettings` object. The same goes for the initial field indicator, too.
```
Comment (Chrispineagunja) [inline ej2-asp-core-mvc/pdfviewer/EJ2_ASP.NETCORE/how-to/change-signature-indicator-text.md:46]: If you want to change these properties, simply modify the corresponding value in the `signatureIndicatorSettings` object. The same goes for the initial field indicator.

**Pull request closed without merging**